services/director-v2/tests/unit/with_dbs/test_modules_dynamic_sidecar_docker_service_specs.py

```diff
@@ -239,6 +239,12 @@ def expected_dynamic_sidecar_spec(
                     "FORWARD_ENV_DISPLAY": ":0",
                     "NODE_PORTS_400_REQUEST_TIMEOUT_ATTEMPTS": "3",
                     "DYNAMIC_SIDECAR_LOG_LEVEL": "DEBUG",
+                    "DY_DEPLOYMENT_REGISTRY_SETTINGS": (
+                        '{"REGISTRY_AUTH": false, "REGISTRY_PATH": null, '
+                        '"REGISTRY_URL": "foo.bar.com", "REGISTRY_USER": '
+                        '"test", "REGISTRY_PW": "test", "REGISTRY_SSL": false}'
+                    ),
+                    "DY_DOCKER_HUB_REGISTRY_SETTINGS": "null",
```

Comment:
I do not see any test with DOCKER_HUB_REGISTRY_SETTINGS set how do you know it works?